tests/es6-export.js

```diff
@@ -136,13 +136,14 @@ describe("export declaration", () => {
         const scope = scopeManager.scopes[1];
 
         expect(scope.type).to.be.equal("module");
-        expect(scope.variables).to.have.length(0);
-        expect(scope.references).to.have.length(1);
+        expect(scope.variables).to.have.length(1);
+        expect(scope.references).to.have.length(2);
         expect(scope.references[0].identifier.name).to.be.equal("x");
+        expect(scope.references[1].identifier.name).to.be.equal("x");
     });
 
     it("should refer exported references#2", () => {
-        const ast = espree("export {v as x};");
+        const ast = espree("const v = 1; export {v as x};");
```

Comment:
More recent versions of Espree will throw an error if the export is not defined: SyntaxError: Export 'v' is not defined